api/v1alpha1/ramenconfig_types.go

```diff
@@ -147,7 +147,8 @@ type RamenConfig struct {
 
 	MultiNamespace struct {
 		// Enables feature to protect resources in namespaces other than VRG's
-		FeatureEnabled bool `json:"FeatureEnabled,omitempty"`
+		FeatureEnabled   bool `json:"FeatureEnabled,omitempty"`
+		VolsyncSupported bool `json:"volsyncSupported,omitempty"`
```

Comment:
Using positive flags is nice but volsync is expected to be supported in the next release, and then we will have to extra work to enable it, or remove it and deal with leftover configs that have no effect.

Should we use VolsyncDisabled to match ramen config? We will have to set this in the config only for this release.

Reply:
Doing so would require changes in the build system and I am trying to avoid such change.